K8s/node taint

WillBeebe opened the pull request:
Closes: https://linear.app/massdriver/issue/ORC-218/augment-create-type-for-k8s-node-taint

mclacore commented on definitions/types/k8s-node-taint.json:
```diff
@@ -0,0 +1,46 @@
+            "type": "string",
+            "title": "Effect",
+            "default": "NO_SCHEDULE",
+            "enum": [
```
Does `enum` work with `const` and `title` pairs? I thought it was just `oneOf`

WillBeebe replied:
Great question! I did run this through `mass bundle build` but let me check if this is valid json schema. I want the const / title thingy here. :)

WillBeebe replied:
Made the fix for `oneOf`, ran that through bundle preview.

mclacore reviewed (changes requested):
Just a question about the `enum` used.

WillBeebe commented on definitions/types/k8s-node-taint.json:
```diff
@@ -0,0 +1,46 @@
+            "type": "string",
+            "title": "Effect",
+            "default": "NO_SCHEDULE",
+            "enum": [
```
```suggestion
            "oneOf": [
```

coryodaniel reviewed (changes requested):
Do you have a link to an RJSF playground to viz?

WillBeebe commented:
> Do you have a link to an RJSF playground to viz?

I can plop stuff into there, I've been using the bundle preview.

WillBeebe commented:
@coryodaniel https://rjsf-team.github.io/react-jsonschema-form/#eyJmb3JtRGF0YSI6eyJlZmZlY3QiOiJOT19TQ0hFRFVMRSIsImZpcnN0TmFtZSI6IkNodWNrcyIsImxhc3ROYW1lIjoiTm9ycmlzIiwiYWdlIjo3NSwiYmlvIjoiUm91bmRob3VzZSBraWNraW5nIGFzc2VzIHNpbmNlIDE5NDAiLCJwYXNzd29yZCI6Im5vbmVlZCIsInRlbGVwaG9uZSI6IjEtODAwLUtJQ0tBU1MifSwic2NoZW1hIjp7IiRzY2hlbWEiOiJodHRwOi8vanNvbi1zY2hlbWEub3JnL2RyYWZ0LTA3L3NjaGVtYSIsInR5cGUiOiJvYmplY3QiLCJ0aXRsZSI6IlRhaW50IiwiZGVzY3JpcHRpb24iOiJBIHRhaW50IHRvIGFwcGx5IHRvIEt1YmVybmV0ZXMgbm9kZXMiLCJyZXF1aXJlZCI6WyJ0YWludF9rZXkiLCJ0YWludF92YWx1ZSIsImVmZmVjdCJdLCJwcm9wZXJ0aWVzIjp7InRhaW50X2tleSI6eyJ0eXBlIjoic3RyaW5nIiwicGF0dGVybiI6Il4oPzpbYS16MC05XXxbYS16MC05XVthLXowLTktXXswLDYxfVthLXowLTldKSQiLCJtZXNzYWdlIjp7InBhdHRlcm4iOiJOYW1lIG11c3QgZm9sbG93IFJGQyAxMTIzOiBhdCBtb3N0IDYzIGNoYXJhY3RlcnMsIG9ubHkgbG93ZXJjYXNlIGFscGhhbnVtZXJpYyBjaGFyYWN0ZXJzIG9yICctJywgYW5kIHN0YXJ0IGFuZCBlbmQgd2l0aCBhbiBhbHBoYW51bWVyaWMgY2hhcmFjdGVyLiJ9LCJkZXNjcmlwdGlvbiI6IiIsInRpdGxlIjoiS2V5IiwiZXhhbXBsZXMiOlsid29ya2xvYWRzIl19LCJ0YWludF92YWx1ZSI6eyJ0eXBlIjoic3RyaW5nIiwicGF0dGVybiI6Il4oPzpbYS16MC05XXxbYS16MC05XVthLXowLTktXXswLDYxfVthLXowLTldKSQiLCJtZXNzYWdlIjp7InBhdHRlcm4iOiJOYW1lIG11c3QgZm9sbG93IFJGQyAxMTIzOiBhdCBtb3N0IDYzIGNoYXJhY3RlcnMsIG9ubHkgbG93ZXJjYXNlIGFscGhhbnVtZXJpYyBjaGFyYWN0ZXJzIG9yICctJywgYW5kIHN0YXJ0IGFuZCBlbmQgd2l0aCBhbiBhbHBoYW51bWVyaWMgY2hhcmFjdGVyLiJ9LCJ0aXRsZSI6IlZhbHVlIiwiZGVzY3JpcHRpb24iOiIiLCJleGFtcGxlcyI6WyJncHUiXX0sImVmZmVjdCI6eyJ0eXBlIjoic3RyaW5nIiwidGl0bGUiOiJFZmZlY3QiLCJkZWZhdWx0IjoiTk9fU0NIRURVTEUiLCJvbmVPZiI6W3siY29uc3QiOiJQUkVGRVJfTk9fU0NIRURVTEUiLCJ0aXRsZSI6IlByZWZlciBObyBTY2hlZHVsZSJ9LHsiY29uc3QiOiJOT19TQ0hFRFVMRSIsInRpdGxlIjoiTm8gU2NoZWR1bGUifSx7ImNvbnN0IjoiTk9fRVhFQ1VURSIsInRpdGxlIjoiTm8gRXhlY3V0ZSJ9XX19fSwidWlTY2hlbWEiOnsiZmlyc3ROYW1lIjp7InVpOmF1dG9mb2N1cyI6dHJ1ZSwidWk6ZW1wdHlWYWx1ZSI6IiIsInVpOmF1dG9jb21wbGV0ZSI6ImZhbWlseS1uYW1lIn0sImxhc3ROYW1lIjp7InVpOmVtcHR5VmFsdWUiOiIiLCJ1aTphdXRvY29tcGxldGUiOiJnaXZlbi1uYW1lIn0sImFnZSI6eyJ1aTp3aWRnZXQiOiJ1cGRvd24iLCJ1aTp0aXRsZSI6IkFnZSBvZiBwZXJzb24iLCJ1aTpkZXNjcmlwdGlvbiI6IihlYXJ0aCB5ZWFyKSJ9LCJiaW8iOnsidWk6d2lkZ2V0IjoidGV4dGFyZWEifSwicGFzc3dvcmQiOnsidWk6d2lkZ2V0IjoicGFzc3dvcmQiLCJ1aTpoZWxwIjoiSGludDogTWFrZSBpdCBzdHJvbmchIn0sInRlbGVwaG9uZSI6eyJ1aTpvcHRpb25zIjp7ImlucHV0VHlwZSI6InRlbCJ9fX0sInRoZW1lIjoiZGVmYXVsdCIsImxpdmVTZXR0aW5ncyI6eyJzaG93RXJyb3JMaXN0IjoidG9wIiwidmFsaWRhdGUiOmZhbHNlLCJkaXNhYmxlIjpmYWxzZSwicmVhZG9ubHkiOmZhbHNlLCJvbWl0RXh0cmFEYXRhIjpmYWxzZSwibGl2ZU9taXQiOmZhbHNlfX0=

WillBeebe commented:
Cleaned up link, removed the UI schema and other placeholder things. Sorry I haven't used this tool much yet.
https://rjsf-team.github.io/react-jsonschema-form/#eyJmb3JtRGF0YSI6eyJlZmZlY3QiOiJOT19TQ0hFRFVMRSJ9LCJzY2hlbWEiOnsiJHNjaGVtYSI6Imh0dHA6Ly9qc29uLXNjaGVtYS5vcmcvZHJhZnQtMDcvc2NoZW1hIiwidHlwZSI6Im9iamVjdCIsInRpdGxlIjoiVGFpbnQiLCJkZXNjcmlwdGlvbiI6IkEgdGFpbnQgdG8gYXBwbHkgdG8gS3ViZXJuZXRlcyBub2RlcyIsInJlcXVpcmVkIjpbInRhaW50X2tleSIsInRhaW50X3ZhbHVlIiwiZWZmZWN0Il0sInByb3BlcnRpZXMiOnsidGFpbnRfa2V5Ijp7InR5cGUiOiJzdHJpbmciLCJwYXR0ZXJuIjoiXig/OlthLXowLTldfFthLXowLTldW2EtejAtOS1dezAsNjF9W2EtejAtOV0pJCIsIm1lc3NhZ2UiOnsicGF0dGVybiI6Ik5hbWUgbXVzdCBmb2xsb3cgUkZDIDExMjM6IGF0IG1vc3QgNjMgY2hhcmFjdGVycywgb25seSBsb3dlcmNhc2UgYWxwaGFudW1lcmljIGNoYXJhY3RlcnMgb3IgJy0nLCBhbmQgc3RhcnQgYW5kIGVuZCB3aXRoIGFuIGFscGhhbnVtZXJpYyBjaGFyYWN0ZXIuIn0sImRlc2NyaXB0aW9uIjoiIiwidGl0bGUiOiJLZXkiLCJleGFtcGxlcyI6WyJ3b3JrbG9hZHMiXX0sInRhaW50X3ZhbHVlIjp7InR5cGUiOiJzdHJpbmciLCJwYXR0ZXJuIjoiXig/OlthLXowLTldfFthLXowLTldW2EtejAtOS1dezAsNjF9W2EtejAtOV0pJCIsIm1lc3NhZ2UiOnsicGF0dGVybiI6Ik5hbWUgbXVzdCBmb2xsb3cgUkZDIDExMjM6IGF0IG1vc3QgNjMgY2hhcmFjdGVycywgb25seSBsb3dlcmNhc2UgYWxwaGFudW1lcmljIGNoYXJhY3RlcnMgb3IgJy0nLCBhbmQgc3RhcnQgYW5kIGVuZCB3aXRoIGFuIGFscGhhbnVtZXJpYyBjaGFyYWN0ZXIuIn0sInRpdGxlIjoiVmFsdWUiLCJkZXNjcmlwdGlvbiI6IiIsImV4YW1wbGVzIjpbImdwdSJdfSwiZWZmZWN0Ijp7InR5cGUiOiJzdHJpbmciLCJ0aXRsZSI6IkVmZmVjdCIsImRlZmF1bHQiOiJOT19TQ0hFRFVMRSIsIm9uZU9mIjpbeyJjb25zdCI6IlBSRUZFUl9OT19TQ0hFRFVMRSIsInRpdGxlIjoiUHJlZmVyIE5vIFNjaGVkdWxlIn0seyJjb25zdCI6Ik5PX1NDSEVEVUxFIiwidGl0bGUiOiJObyBTY2hlZHVsZSJ9LHsiY29uc3QiOiJOT19FWEVDVVRFIiwidGl0bGUiOiJObyBFeGVjdXRlIn1dfX19LCJ1aVNjaGVtYSI6eyJ1aTpvcmRlciI6WyIqIl19LCJ0aGVtZSI6ImRlZmF1bHQiLCJsaXZlU2V0dGluZ3MiOnsic2hvd0Vycm9yTGlzdCI6InRvcCIsInZhbGlkYXRlIjpmYWxzZSwiZGlzYWJsZSI6ZmFsc2UsInJlYWRvbmx5IjpmYWxzZSwib21pdEV4dHJhRGF0YSI6ZmFsc2UsImxpdmVPbWl0IjpmYWxzZX19

mclacore reviewed (approved):
LGTM

chrisghill reviewed (approved):
Looks good!
